src/pint/models/parameter.py

```diff
@@ -109,9 +109,11 @@ def __init__(
         get_value=lambda x: x,
         prior=priors.Prior(priors.UniformUnboundedRV()),
         set_uncertainty=fortran_float,
+        repeatable=False
```

Comment:
Why do you need a `repeatable` attribute? Isn't it the case that only `maskParameter`s are repeatable? You could just set that in `__init__` - False for the  base class and True in `maskParameter.__init__`.